publish/ictrl_be.spec

```
@@ -8,7 +8,8 @@ block_cipher = None
 a = Analysis(['../ictrl_be.py'],
              pathex=['.'],
              binaries=[],
-             datas=[('../client/build', './client')],
+             datas=[('../client/build', './client'),
+                    ('../log_config.yaml', '..')]
```

Sorry I should have been more clear about how to validate the changes - we should `cd <iCtrl-root>; .\publish\win64_build.bat` and observe the build to be successful except the GITHUB publish step.

I don't think the current configuration is in valid syntax and performs what we wish to achieve. Let's update and validate.
```suggestion
             datas=[('../client/build', './client'),
                    ('../log_config.yaml', '..')],
```
1. Add missing comma.
2. Copy the log config file into the packaged application's main directory (the 2nd element in the tuple should be `.` instead of `..`).